Packs/cisco-meraki/Integrations/ciscomerakiv2/ciscomerakiv2.py

```diff
@@ -0,0 +1,515 @@
+        outputs_prefix='Meraki.DeviceRemoval',
+        outputs=outputs,
+        outputs_key_field=['networkId', 'serial'],
+        readable_output=readable_output
+    )
+    return_results(command_results)
+
+
+def meraki_fetch_ssids_command(client: Client, args: dict):
+    network_id = args.get('networkId')
+    res = client.get_network_ssids(network_id=network_id)
+    command_results = CommandResults(
+        outputs_prefix='Meraki.SSID',
+        outputs=res,
+        outputs_key_field=['number', 'name'],
+        readable_output=tableToMarkdown(f'Network ({network_id}) SSIDs:', res)
+    )
+    return_results(command_results)
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    """main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+    params = demisto.params()
+    args = demisto.args()
+    api_key = params.get('apiKey')
+
+    verify = not params.get('insecure')
+    proxy = params.get('proxy')
+    headers = {
+        'X-Cisco-Meraki-API-Key': api_key,
+        'Content-Type': 'application/json',
+    }
+
+    commands = {
+        'meraki-fetch-organizations': meraki_fetch_org_command,
+        'meraki-fetch-organization-devices': meraki_fetch_org_devices_command,
+        'meraki-get-organization-license-state': meraki_get_org_lic_state_command,
+        'meraki-fetch-networks': meraki_fetch_networks_command,
+        'meraki-fetch-network-devices': meraki_fetch_network_devices_command,
+        'meraki-update-device': meraki_update_device_command,
+        'meraki-claim-device': meraki_claim_device_command,
+        'meraki-fetch-device': meraki_fetch_device_command,
+        'meraki-fetch-organization-uplinks': meraki_fetch_organization_uplinks_command,
+        'meraki-fetch-device-clients': meraki_fetch_device_clients_command,
+        'meraki-fetch-network-clients': meraki_fetch_network_clients_command,
+        'meraki-search-clients': meraki_search_clients_command,
+        'meraki-fetch-wireless-firewall-rules': meraki_fetch_wireless_firewall_rules_command,
+        'meraki-fetch-appliance-firewall-rules': meraki_fetch_appliance_firewall_rules_command,
+        'meraki-update-wireless-firewall-rules': meraki_update_wireless_firewall_rules_command,
+        'meraki-update-appliance-firewall-rules': meraki_update_appliance_firewall_rules_command,
+        'meraki-remove-device': meraki_remove_device_command,
+        'meraki-fetch-ssids': meraki_fetch_ssids_command
+    }
+
+    # try:
+    client = Client(
+        base_url,
+        verify=verify,
+        proxy=proxy,
+        headers=headers
+    )
+
+    command = demisto.command()
+
+    if command == 'test-module':
+        test_module_command(client)
+
+    elif command in commands:
+        commands[command](client, args)
+
+    # except Exception as err:
+    #    demisto.error(traceback.format_exc())  # print the traceback
+    #    return_error(f'Failed to execute {command} command.\nError:\n{str(err)}')
```

Comment:
please resume the try/except block 
as it will catch any exception and will add log entry for that

in addition please add condition for non implemented command
```
    if command == 'test-module':
        test_module_command(client)

    elif command in commands:
        commands[command](client, args)
        
     else:
         raise NotImplementedError(f"command {command} is not implemented.")
```